Core/BookmarksCleanupErrorHandling.swift

```diff
@@ -26,11 +26,15 @@ public class BookmarksCleanupErrorHandling: EventMapping<BookmarksCleanupError>
 
     public init() {
         super.init { event, _, _, _ in
-            let domainEvent = Pixel.Event.bookmarksCleanupFailed
-            let processedErrors = CoreDataErrorsParser.parse(error: event.cleanupError as NSError)
-            let params = processedErrors.errorPixelParameters
+            if event.cleanupError is BookmarksCleanupCancelledError {
```

Comment:
Here is an update to cleaner error handling that I forgot to include in the original release :)